# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../sfdisk/sfdisk.001.include_sys_syscalls_h.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Imported from buildroot
http://git.uclibc.org/buildroot/tree/package/sfdisk

--- sfdisk/sfdisk.c.oorig	2006-10-09 16:24:06.000000000 +0200
+++ sfdisk/sfdisk.c	2006-10-09 16:24:20.000000000 +0200
@@ -47,6 +47,9 @@
 #include <sys/stat.h>
 #include <sys/utsname.h>
 #ifdef __linux__
+#define _LIBC
+#include <sys/syscall.h>
+#undef _LIBC
 #include <linux/unistd.h>	/* _syscall */
 #endif
 #include "nls.h"
